fitPanel/controller/SimpleFitPanel.controller.js

```diff
@@ -72,14 +72,18 @@ sap.ui.define([
       },
 
       selectRB: function(){
+
         var lib = this.getView().getModel().getData().fLibrary;
 
+        if (this.websocket)
+            this.websocket.Send(this.getView().getModel().getData().lib);
+
         if(lib == 0){
```

Comment:
You should not send this information to server side.
All changes in items list of combo boxes should be handled in JavaScript
Only when user press "Fit", configured values (all together) will be send to the C++ and fitting will be performed.
I will merge your code, but also change it immediately